model/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import find_packages
+from setuptools import setup
+
+with open('requirements.txt') as f:
```

Comment:
should we commit to single or double quotes everywhere ? PEP8 does not make a recommandation (only to use double quotes for multiline strings)

personally I use double quotes but we can decide what we want...

```suggestion
with open("requirements.txt") as f:
```